js/src/main/scala/org/portablescala/reflect/Reflect.scala

```diff
@@ -74,6 +82,7 @@ object Reflect {
    *  @param loader
    *    Ignored
    */
+  @silent("never used")
```

Comment:
If it is not used, we should add a test.

Reply:
That was just about the unsued parameter. I've made it more explanatory.